core/src/io/anuke/mindustry/world/blocks/power/LiquidGenerator.java

```diff
@@ -11,6 +11,7 @@
 import io.anuke.ucore.graphics.Draw;
 import io.anuke.ucore.util.Mathf;
 
+// TODO figure out what to do with this
```

Comment:
Does this refer to the fact that these generators output to buffers, or something else?

Reply:
Yes. `PowerGraph` currently ignores generators' buffers, so these don't generate any power now.

Reply:
This could be fixed by assigning an entity `currentOutput` variable each update(), then returning that in `getPowerProduction()`